content/getting_started/UsingEnzyme.md

```diff
@@ -5,6 +5,13 @@ draft: false
 weight: 10
 ---
 
+---
```

Comment:
This shouldn't be repeated?

Reply:
Yes, this should absolutely not get repeated!

Reply:
Solved it with the patch below